packages/venia-ui/lib/RootComponents/Category/category.js

```diff
@@ -47,25 +65,18 @@ const Category = props => {
         }
     }, [filterClear]);
 
-    // run the category query
+    // Re-run the query whenever its variable values change.
     useEffect(() => {
-        setLoading(true);
-        runQuery({
-            variables: {
-                currentPage: Number(currentPage),
-                id: Number(id),
-                idString: String(id),
-                onServer: false,
-                pageSize: Number(pageSize)
-            }
+        refetch({
```

Comment:
I think this is another example of where we should use `useLazyQuery`.

Reply:
👍 https://github.com/magento/pwa-studio/pull/1827/commits/62084a630af4cf3c6b6203612cb84cc6da747cc0